extensions/standard-processors/tests/integration/SecureSocketGetTCPTest.cpp

```diff
@@ -117,23 +117,16 @@ class SecureSocketTest : public IntegrationBase {
     assert(0 == server_socket_->initialize());
 
     isRunning_ = true;
-    check = [this]() -> bool {
-      return isRunning_;
-    };
-    handler = [](std::vector<uint8_t> *b, int *size) {
+    auto handler = [](std::vector<uint8_t> *b) {
       std::cout << "oh write!" << std::endl;
```

Comment:
This is an old debugging comment checked in by accident, but we might as well remove it now.
```suggestion
```

Reply:
removed in e8f2c49